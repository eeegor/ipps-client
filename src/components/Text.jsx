import React from 'react';

export const Text = (props) => {
	const { text } = props;
	return <p className="text">{text}</p>
}