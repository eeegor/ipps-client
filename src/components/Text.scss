.text {
  line-height: 1.25rem;
}
