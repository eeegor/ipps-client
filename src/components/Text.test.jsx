import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import { Text } from './Text';

describe('Text', () => {
  it('renders markup correctly', () => {
    const wrapper = <Text>Hello</Text>;
    const tree = renderer.create(wrapper).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should adopt a text', () => {
    const wrapper = shallow(<Text text="custom-text">Hello</Text>);
    expect(wrapper.contains('custom-text')).toEqual(true);
  });
});
