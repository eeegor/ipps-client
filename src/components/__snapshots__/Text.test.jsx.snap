// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text renders markup correctly 1`] = `
<p
  className="text"
>
  Hello
</p>
`;
